test/index.test.js

```diff
@@ -9,6 +9,44 @@ const { createServer } = require('net')
 const trampolinePath = getDesktopTrampolinePath()
 const run = promisify(execFile)
 
+async function startTrampolineServer(output, stdout = '', stderr = '') {
+  const server = createServer(socket => {
+    socket.pipe(split2(/\0/)).on('data', data => {
+      output.push(data.toString('utf8'))
+    })
+
+    const buffer = Buffer.alloc(2, 0)
+
+    // Send stdout
+    buffer.writeUInt16LE(stdout.length, 0)
```

Comment:
`writeUInt16LE` assumes the trampoline (and tests) will always run in little-endian machines. In the unlikely scenario where we support big-endian architectures, we should revisit this (or the trampoline, to make sure it always reads the bytes in the right order).